doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-background-instance-con.adoc

```diff
@@ -0,0 +1,4 @@
+[id='test-scenarios-background-instance-con']
+= Background instance in test scenarios
+
+In test scenario designer, you can use the *Background* tab to add and set the background data for rules-based as well as DMN based test scenarios. You can add and define the GIVEN data which is common for the entire test scenario simulation, based on the available data objects. *Background* tab has the ability to add and share the data among every test scenario. The data which is added in the *Background* tab will not be overridden with the *Model* tab data. 
```

Comment:
Question - is this the correct description of the mechanism:

"The data which is added in the *Background* tab"

Is it "in the", "using the", "to the" or "with the"?